docs/using-jda/interactions.md

```diff
@@ -96,64 +97,116 @@ When you use `deferReply` the first message sent to this webhook will act identi
+       } else if (event.getName().equals("info")) {
+           event.reply("Click the buttons for more info")
+               .addActionRow( // link buttons don't send events, they just open a link in the browser when clicked
+                   Button.link("https://github.com/DV8FromTheWorld/JDA", "GitHub")
+                     .withEmoji(Emoji.fromMarkdown("<:github:849286315580719104>")), // Link Button with label and emoji
+                   Button.link("https://ci.dv8tion.net/job/JDA/javadoc/", "Javadocs")) // Link Button with only a label
+               .queue();
+       }
+   }
+
+   @Override
+   public void onButtonClick(ButtonClickEvent event) {
+       if (event.getComponentId().equals("hello")) {
+           event.reply("Hello :)").queue(); // send a message in the channel
+       } else if (event.getComponentId().equals("emoji")) {
+           event.editMessage("That button didn't say click me").queue(); // update the message
+       }
+   }
+ }
+```
+
+## Selection Menus (Dropdowns)
+
+Every dropdown can be disabled and have up to 25 options.
+
+It's possible to set the minimum and maximum amount of options to be selected.
+
+Each option can have its own label, description, and emoji.
+There can be multiple options selected and set as default.
+
+![Example Selection Menu With A Default Value](https://i.imgur.com/44q006n.png)
+
+### Handling SelectionMenuEvent
+
+When a user selects their options from a dropdown and submits their choices, you will receive a `SelectionMenuEvent` for the respective interaction with the selected values.
 
-      @Override
-      public void onButtonClick(ButtonClickEvent event) {
-          if (event.getComponentId().equals("hello")) {
-              event.reply("Hello :)").queue(); // send a message in the channel
-          } else if (event.getComponentId().equals("emoji")) {
-              event.editMessage("That button didn't say click me").queue(); // update the message
-          }
-      }
-    }
-    ```
\ No newline at end of file
+!!! example
+
+```java
+ public class DropdownBot extends ListenerAdapter {
+	@Override
+	public void onSlashCommand(SlashCommandEvent event) {
+		if (event.getName().equals("food")) {
+			event.reply("Choose your favorite food")
+				.addActionRow(
+					SelectionMenu.create("choose-food")
+					  .addOption("Pizza", "pizza", "Classic") // SelectOption with only the label, value, and description
+					  .addOptions(SelectOption.of("Hamburger", "hamburger") // another way to create a SelectOption
+							.withDescription("Tasty") // this time with a description
+							.withEmoji(Emoji.fromUnicode("\uD83C\uDF54")) // and an emoji
+							.withDefault(true)) // while also being the default option
+					.build())
+				.queue();
+		}
+	}
+  
+	@Override
+	public void onSelectionMenu(SelectionMenuEvent event) {
+		if (event.getComponentId().equals("choose-food")) {
+			event.reply("You chose " + event.getValues().get(0)).queue();
+		}
+	}
+ }
+```
```

Comment:
This code block also needs 4 spaces of indentation to fit inside the admonition

Reply:
This is indented with Tabs.
Please convert the tabs to spaces. We should not have tabs in the project. If there are any tabs throughout the project we should convert them to spaces @Aikovdp

Reply:
I've checked, and it looks like there aren't any tabs anywhere, they were only added in this PR. Once Cane replaces his tabs with spaces we'll be all good.